src/main/java/ch/njol/skript/variables/VariablesStorage.java

```diff
@@ -119,7 +125,23 @@ protected VariablesStorage(String name) {
 					// Ignored as the `closed` field will indicate whether the thread actually needs to stop
 				}
 			}
-		}, "Skript variable save thread for database '" + name + "'");
+		}, "Skript variable save thread for database '" + type + "'");
+	}
+
+	/**
+	 * Get the config name of a database
+	 * @return name of database
```

Comment:
Can a note be added here that the value comes from whatever the user defined as the section node from config.sk, anything will do, as long as a developer knows it's not the type.